packaging/windows/hostfxr/registrykeys.wxs

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
```

Comment:
Not sure we need this file, the registry key should be set by the muxer, I think.